infra/cifuzz/cifuzz.py

```diff
@@ -292,6 +294,8 @@ def check_fuzzer_build(out_dir, sanitizer='address'):
       '-e',
       'ARCHITECTURE=' + DEFAULT_ARCHITECTURE,
       '-e',
+      'CIFUZZ=True',
+      '-e',
       'FUZZING_LANGUAGE=c++',  # FIXME: Add proper support.
```

Comment:
Unrelated note, please fix this FUZZING_LANGUAGE sometime, we want it to work across all languages.